src/Splash.js

```diff
@@ -8,24 +8,30 @@ export default class Splash extends Component {
   displayMainPage = (event) => {
     event.preventDefault();
     let splashPage = document.querySelector('.splash')
-    let displayNone = document.querySelectorAll('.display-none')
+    let fetchDogs = document.querySelector('.fetch-dogs-btn')
+    let cardContainer = document.querySelector('.card-container-selector')
     splashPage.classList.remove('splash')
     splashPage.classList.add('display-none')
-    displayNone.forEach(element => {
-      element.classList.remove('displayNone')
-      element.classList.add('display-block')
-    })
+    cardContainer.classList.remove('display-none')
+    cardContainer.classList.add('card-container')
```

Comment:
Check your other PR for my notes on doing conditional classes on DOM elements: https://stackoverflow.com/questions/30533171/react-js-conditionally-applying-class-attributes

You really shouldn't be doing **any** `document.querySelectors()` or `document.getElementByIds()`.  That is vanilla javascript DOM manipulation, which takes away from the automatic power of React. Any instances where you're using these vanilla JS document methods is probably a sign you're doing something wrong. If you have a difficult time removing these, let's do a check-in during the next work time session and go over that.